### PR TITLE
debug : collect tpm event log

### PR DESCRIPTION
**Pull request by shjala**

### CODE REVIEW

File: pkg/debug/scripts/collect-info.sh
Comment (erkki) [inline pkg/debug/scripts/collect-info.sh:1]: Recommend checking for the presence of the file and tpm0 folder. In some cases the eventlog shall not exist while go-tpm still works.

File: pkg/debug/scripts/collect-info.sh [hunk 1]
```diff
@@ -144,6 +144,9 @@ qemu-affinities.sh    > "$DIR/qemu-affinities"
 echo "- iommu groups"
 iommu-groups.sh       > "$DIR/iommu-groups"
 
+echo "- TPM event log"
+cat /sys/kernel/security/tpm0/binary_bios_measurements > "$DIR/tpm_event_log"
```
Comment (eriknordmark) [inline pkg/debug/scripts/collect-info.sh:148]: Shouldn't we check for the existence of the file before doing the copy?
Also, how large can the event log be? The $DIR is backed by memory which is why we symlink in other parts.

Comment (rouming) [reply pkg/debug/scripts/collect-info.sh]: Quickly found here: https://trustedcomputinggroup.org/wp-content/uploads/TCG_ACPIGeneralSpecification-Family-1.2-and-2.0-Ver1.2-Rev8_public-revie....pdf

```
Log Area Minimum
Length (LAML)

Identifies the minimum length (in bytes) of the system’s preboot TCG event log area.
Note: The “PC Client conventional BIOS” specification defines a minimum log size of 64KB."

Log Area Start
Address (LASA)

Note: The log area ranges from address LASA to
LASA+(LAML-1).
Note: The format of the TCG event log area is defined in the “PC Client PFP” specification, Section 9. The crypto agile log format as defined by
```

Though no max value. Probably @shjala has more exact info on that.

Comment (shjala) [reply pkg/debug/scripts/collect-info.sh]: I think it can get arbitrary big, changed to symlink.

**Pull request merged**